homeassistant/components/overkiz/sensor.py

```diff
@@ -92,7 +92,7 @@ class OverkizSensorDescription(SensorEntityDescription):
         name="Water volume estimation at 40 °C",
         icon="mdi:water",
         native_unit_of_measurement=UnitOfVolume.LITERS,
-        device_class=SensorDeviceClass.VOLUME,
+        device_class=SensorDeviceClass.WATER,
```

Comment:
I am not sure about this one.
`WATER` device class relates to water meters - not to a volume in a tank.

Reply:
You are correct, this actually is a volume, not the consumption.

Reply:
According to the end user, it’s look like we can read it as a comsumption device: https://github.com/home-assistant/core/issues/83736

Reply:
According to the end user, it’s look like we can read it as a comsumption device: https://github.com/home-assistant/core/issues/83736

@MassimoL06 Can you confirm?

Reply:
I think he has the other sensor device? Or am I missing something.